stories/Molecules/Blocks/CountryCard/country-card.scss

```diff
@@ -87,8 +87,7 @@
               &__caption {
                 bottom: 0;
                 opacity: 1;
-                border-left: 1px solid $color-white;
-                border-right: 1px solid $color-white;
+                width: calc(100% + 2px);
```

Comment:
@Shamimkhan321 This will add 2px additional width to left and right which will go beyond the container width. Can we handle this in some other way?